index.html

```diff
@@ -219,8 +219,10 @@
             <br>
             <br>
         </form>
-    	<button type="submit" class="btn btn-primary mb-2" id="makeNewQ" onClick=CreateNewQuota_DOM(document.getElementById("QuotaRows").value) >Create New Quota</button>
-
+        <button type="submit" class="btn btn-primary mb-2" id="makeNewQ" onClick=CreateNewQuota_DOM(document.getElementById("QuotaRows").value) >Create New Quota</button>
+        <button type="submit" class="btn btn-primary mb-2" id="exportQuotas" onClick=ExportQuotas() >Export Quotas</button>
```

Comment:
Can you create the export button only if the import has occurred OR the sample has been flagged?